python/hpx/process.py

```diff
@@ -14,12 +14,12 @@
 from threading import Thread, Lock
 from time import sleep, time
 from subprocess import Popen, STDOUT, PIPE
-from types import StringType
+#from types import StringType
 from shlex import split
 from signal import SIGKILL
 from os import kill
 from platform import system
-from Queue import Queue, Empty
+from queue import Queue, Empty
```

Comment:
How about writing this as
```
import sys
sys.version_info[0] < 3:
    from Queue import Queue, Empty
else:
    from queue import Queue, Empty
```
instead?

Reply:
I am not sure why this works for python 2.7 and python 3 in the current version.